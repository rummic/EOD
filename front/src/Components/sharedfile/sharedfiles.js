import React, { Component } from "react";





class sharedfiles extends Component {
    constructor(props) {
        super(props);
        this.state = {
            id: null,
          value: "",
        };
      }
    
      componentDidMount() {
        fetch("https://localhost:44388/api/Documents/DocumentSeen"+ window.location.search,{
            method: "PATCH",
            headers: {
              "Content-Type": "application/json",
            },})
          .then(response => response.json())
          .then(parseJSON => {
            this.setState({
                value: parseJSON.value
              });
          });

      }
    render() {
        console.log(window.location.search)
        console.log(this.state.value)
        window.open('https://localhost:44388/Documents/'+ this.state.value, "_blank")
      return (
        null
      );
    }
  }
  
  export default sharedfiles;