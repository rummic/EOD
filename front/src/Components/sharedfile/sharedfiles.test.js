import React from "react";
import sharedfiles from "./sharedfiles";
import { shallow } from 'enzyme';

describe('sharedfiles', () => {
    test('Rendering shaderfiles component', () => {
        const wrapper = shallow(<sharedfiles />);
        expect(wrapper).toMatchSnapshot();
    });
});  