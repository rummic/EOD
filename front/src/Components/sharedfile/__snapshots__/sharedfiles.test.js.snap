// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sharedfiles Rendering shaderfiles component 1`] = `ShallowWrapper {}`;
